Dotfiles/claude/skills/wtf-explainer/assets/template/js/main.js

```
@@ -0,0 +1,338 @@
+  followBox.addEventListener('change', function () { follow = followBox.checked; });
+  function setFollow(v) { follow = v; followBox.checked = v; }
+
+  document.addEventListener('keydown', function (e) {
+    if (e.target.tagName === 'INPUT') return;
+    switch (e.key.toLowerCase()) {
+      case ' ': e.preventDefault(); Sim.toggle(); UI.paint(true); break;
+      case 's': Sim.step(); break;
+      case 'r': UI.resetAll(); break;
+      case 'f': setFollow(!follow); break;
+      case 'l':
+        var lb = document.getElementById('labels');
```

Biome (2.5.6)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)